hazelcast/src/main/java/com/hazelcast/spi/MutatingOperation.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi;
+
+/**
+ * Marker interface for operations that changes map state/data.
+ * Used for quorum to reject operations if quorum size not satisfied
+ */
+public interface MutatingOperation {
```

Comment:
Don't think we need this interface. BackupAwareOperation already is indication that operation can mutate state.